python/semantic_kernel/agents/agent.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import uuid
+from abc import ABC, abstractmethod
+
+from semantic_kernel.agents.agent_channel import AgentChannel
+from semantic_kernel.kernel_pydantic import KernelBaseModel
+from semantic_kernel.utils.experimental_decorator import experimental_class
+
+
+@experimental_class
+class Agent(ABC, KernelBaseModel):
```

Comment:
should this be called `AgentBase` or `KernelAgentBase`? that would be more inline with the rest

Reply:
We can do AgentBase.